src/pageTransition.js

```diff
@@ -193,7 +193,7 @@ import pageTransitionClickContentScript from "include:./content-scripts/pageTran
  * @property {boolean} privateWindows - Whether to notify the listener about page transitions in
  * private windows and whether to consider pages loaded in private windows when generating
  * time-based transition information.
- * @property {browser.contentScripts.RegisteredContentScript} contentScript - The content
+ * @property {browser.scripting.RegisteredContentScript} contentScript - The content
```

Comment:
same as `linkExposure` above, this should remain `contentScripts`

Reply:
Oh thanks yeah blind search/replace on my part, let's leave these alone until they are ported.